openvdb_houdini/houdini/SOP_NodeVDB.cc

```diff
@@ -588,23 +587,42 @@ class SESIOpenVDBOpPolicy: public houdini_utils::OpPolicy
 };
 
 
-/// @brief OpPolicy for OpenVDB operator types at DWA
-class DWAOpenVDBOpPolicy: public houdini_utils::DWAOpPolicy
+/// @brief Default OpPolicy for OpenVDB operator types
+class SESIOpenVDBOpPolicy: public DefaultOpenVDBOpPolicy
 {
 public:
-    /// @brief OpenVDB operators of each flavor (SOP, POP, etc.) share
-    /// an icon named "SOP_OpenVDB", "POP_OpenVDB", etc.
-    std::string getIconName(const houdini_utils::OpFactory& factory) override
+    std::string getName(const houdini_utils::OpFactory& factory, const std::string& english) override
     {
-        return factory.flavorString() + "_OpenVDB";
+        std::string name = DefaultOpenVDBOpPolicy::getName(factory, english);
+        UT_String s(name);
+        // Lowercase
+        s.toLower();
+        return s.toStdString();
+    }
+};
+
+
+/// @brief ASWF OpPolicy for OpenVDB operator types
+class ASWFOpenVDBOpPolicy: public DefaultOpenVDBOpPolicy
+{
+public:
+    std::string getName(const houdini_utils::OpFactory& factory, const std::string& english) override
+    {
+        std::string name = DefaultOpenVDBOpPolicy::getName(factory, english);
+        return "DW_Open" + name;
+    }
+
+    std::string getLabelName(const houdini_utils::OpFactory& factory) override
```

Comment:
For me, at least, GCC 4.8 complains about this:
```
openvdb_houdini/houdini/SOP_NodeVDB.cc:615:17: error: ‘std::string openvdb_houdini::{anonymous}::ASWFOpenVDBOpPolicy::getLabelName(const OpFactory&)’ marked override, but does not override
     std::string getLabelName(const OpFactory& factory) override
                 ^
```
That's nonsense, but I don't know what fix to suggest.